app/components/Toast/ToastContainer.js

```diff
@@ -47,7 +47,8 @@ function toastStyleFactory(index, style) {
   }
   return {
     ...style,
-    bottom: `${2 + index * 4}rem`
+    bottom: `${2 + index * 4}rem`,
+    zIndex: 2
```

Comment:
Add zIndex to the CSS for mobile (line 37 to 45) too. :smile:

Reply:
done